### DISCUSSION TITLE
Is there Any Open source LLM suitable for this?

### DISCUSSION THREAD

**Discussion by mikeyang01** [ElliottDyson/OSGA#16] [Q&A]
looking for best LLM or lora for agents

**Comment by ElliottDyson** [comment]
> looking for best LLM or lora for agents

I have personally been mostly testing with MythoMax Q5_k_m, but I currently plan on forming an instruction tuning dataset to fine-tune models with for more predictable and higher quality outputs

**Reply by mikeyang01** [reply]
Great! waiting for your good news!
